### DISCUSSION TITLE
D-Sides Icon Bounce

### DISCUSSION THREAD

**Discussion by Memelord111** [JordanSantiagoYT/FNF-JS-Engine#344] [Ideas]
Can You Add The D Sides Icon Bounce? On 1 beat they bop there head down and the next they dont incase you forgot how it is

**Comment by JordanSantiagoYT** [comment]
yeah i've checked the d-sides mod and that's not a real thing. what you might be referring to is a script

**Comment by Memelord111** [comment]
yeah nvm its already in the m i engine just by a diffrent name also i was using a mod folder pack and that has it so it explains that
